src/app/upgrade.cpp

```diff
@@ -37,6 +37,22 @@
 
 namespace
 {
+    void storeLegacyDefaults()
+    {
+        const QVariantHash legacyDefaults {
+            {QLatin1String {"BitTorrent/Session/QueueingSystemEnabled"}, true}
```

Comment:
Are there some other settings we should handle here?

Reply:
Maybe different port on each startup thing? Old user who never touched port settings might lose port forwarding and never notice as it’s disabled by default.